src/main/kotlin/org/rust/ide/inspections/RsNamingInspection.kt

```diff
@@ -100,6 +97,11 @@ open class RsSnakeCaseNamingInspection(elementType: String) : RsNamingInspection
         }
         return Pair(false, if (str.isEmpty()) "snake_case" else name.toSnakeCase(false))
     }
+
+    override fun isSuppressedFor(element: PsiElement): Boolean {
+        val function = element.parent as? RsFunction ?: return false
```

Comment:
Let's invoke `super.isSuppressedFor(element)` to check if this inspection is already suppressed for this element like we do in `org.rust.ide.inspections.RsLintInspection#isSuppressedFor`. Otherwise, it prevents suppressing, for example, via comment:

```rust
//noinspection RsFunctionNaming
fn Bar() {}
```

Reply:
BTW maybe move this check into `RsFunctionNamingInspection`? Because it makes no sense for other inheritors

Reply:
I didn't notice that a function-specific inspection exists, thanks.